### ISSUE TITLE
Tor Browser Window Button Colors

### ISSUE THREAD

**Issue by frostbytxs** [Rectify11/Installer#602]
### Description

When hovering over window buttons, the button background goes cyan when Rectify11 is installed.

### Steps To Reproduce

Open Tor Browser, then hover over the window buttons.

### Rectify11 Version

Version 3.2.70.1

### Windows Version

Windows 11 Pro 23H2 22631.4037

### Logs

![image](https://github.com/user-attachments/assets/0d6ccf49-8270-4a01-ac70-972d66fc7f32)
![image](https://github.com/user-attachments/assets/156fcc9c-fbaf-4d98-849f-aa94371c52a8)
![image](https://github.com/user-attachments/assets/efc17cd4-91de-4506-ac70-ae89e8949adf)

**Comment by RaphProductions** [comment]
If it applies to Tor, it should applies to Firefox & it's other forks too?

**Comment by frostbytxs** [comment]
> If it applies to Tor, it should applies to Firefox & it's other forks too?

Nope. At least not in any official Firefox browsers. Haven't tried any of the forked/hardened ones though, might try them later.

**Comment by ooTruffle** [comment]
> If it applies to Tor, it should applies to Firefox & it's other forks too?

This was fixed in firefox awile ago tor is just on a older version

In tor apply [This](https://mrotherguy.github.io/firefox-csshacks/?file=window_control_fallback_for_custom_windows_theme.css) css to fix that issue untill tor updates to fix that issue

**Comment by ooTruffle** [comment]
> If it applies to Tor, it should applies to Firefox & it's other forks too?

Tor is based on a older ff esr release
